app/views/profiles/import/_form.html.haml

```diff
@@ -0,0 +1,5 @@
+= form_with model: @import_data, url: profile_import_index_path, method: :post, class: "ui fluid form", local: true do |f|
+  .required.field
+    = f.file_field :upload, aria: { describedby: "Upload file" }
+
+  = f.submit "Import", class: "ui positive right floated submit button"
```

Comment:
Too much cognitive load on the user, it should be a single button task :P

Reply:
you're not allowed to go to product meetings anymore

Reply:
Problem is that when I add in firefox and chrome imports, that'll be another two buttons. Whereas I could just have a job that is smart enough to determine how to parse the file instead, also opening the ability for any additional custom formats like a csv or something because why not. This single input field is also how pinboard does it, which is what I modeled this off of.